services/ui-src/src/components/fields/RadioField.tsx

```diff
@@ -149,6 +150,8 @@ export const RadioField = (props: PageElementProps<RadioTemplate>) => {
         return;
       case "qmDeliveryMethodChange":
         changeDeliveryMethods(currentPageId, value);
+        // Update zustand state with latest form values
+        setAnswers(form.getValues());
```

Comment:
Does autosave still run after this occurs? That's the other part that is wrapped into the PageWrapper

Reply:
Good catch, during testing I was going through the flow pretty fast and was able update the answers before the `ReportPageWrapper.tsx` autosave was called. Fixed.